diff --git a/single-include/meson.build b/single-include/meson.build
index ee58adf..decf8b2 100644
--- a/single-include/meson.build
+++ b/single-include/meson.build
@@ -2,7 +2,9 @@
 # meson.build here when generating the single file header so that it is placced
 # in the correct location.
 
-prog_python = find_program('python')
+pymod = import('python')
+prog_python = pymod.find_installation()
+message('did this work???? I don t know')
 
 single_main_file = files('CLI11.hpp.in')
 
