--- a/meson.build	2025-01-13 21:23:59
+++ b/meson.build	2025-01-13 21:24:36
@@ -109,13 +109,17 @@
   margs += '-DHAVE_BZIP2'
 endif
 
-lzma_dep = dependency('liblzma', required: get_option('lzma'))
+# CHUMP: appending -NOPE to create an apocryphal dependency name
+# CHUMP: this is done to circument meson from located unwanted libraries on local system
+# CHUMP: (e.g., finding non-universal-binary homebrew libraries that break the build)
+lzma_dep = dependency('liblzma-NOPE', required: get_option('lzma'))
 if lzma_dep.found()
   sources += files('mz_strm_lzma.c')
   margs += '-DHAVE_LZMA'
 endif
 
-zlib_dep = dependency('zlib-ng', required: get_option('zlib'))
+# CHUMP: (see comment above regarding -NOPE)
+zlib_dep = dependency('zlib-ng-NOPE', required: get_option('zlib'))
 if not zlib_dep.found()
   zlib_dep = dependency('zlib', required: get_option('zlib'))
   if zlib_dep.found()
@@ -127,7 +131,8 @@
   margs += '-DHAVE_ZLIB'
 endif
 
-zstd_dep = dependency('libzstd', required: get_option('zstd'))
+# CHUMP: (see comment above regarding -NOPE)
+zstd_dep = dependency('libzstd-NOPE', required: get_option('zstd'))
 if zstd_dep.found()
   sources += files('mz_strm_zstd.c')
   margs += '-DHAVE_ZSTD'
